addons/website/static/src/scss/website.ui.scss

```diff
@@ -51,6 +51,7 @@ body.o_connected_user {
     top: 0;
     margin-top: $o-navbar-height!important;
     opacity: 0;
+    pointer-events:none;
```

Comment:
Missing space after `:` + add a blank line after the property (before nested rules)